theorems/T000180.md

```diff
@@ -1,14 +1,15 @@
 ---
 uid: T000180
 if:
-  P000067: true
+  P000015: true
 then:
   P000061: true
 refs:
-- mr: 2247908
-  name: Products of cozero complemented spaces
+  - zb: "0684.54001"
+    name: General Topology (Engelking, 1989)
 ---
 
-As a set is closed if and only if it is cozero complemented.
-
-Given as Proposition 1.3 of {{mr:2247908}}.
+In a {P15} space, every open set is a cozero set
+(follows for example from Corollary 1.5.13 in {{zb:0684.54001}}).
```

Comment:
We've established in our definition for P15 that perfectly normal is characterized by all closed sets are cozero. Should we add that all open sets are cozero there?